src/app/src/main/kotlin/org/vaccineimpact/api/app/ErrorHandler.kt

```diff
@@ -13,40 +14,50 @@ import spark.Response
 import spark.Spark as spk
 
 @Suppress("RemoveExplicitTypeArguments")
-open class ErrorHandler(private val serializer: Serializer = MontaguSerializer.instance)
+open class ErrorHandler(private val logger: Logger = LoggerFactory.getLogger(ErrorHandler::class.java),
+                        private val postgresHandler: PostgresErrorHandler = PostgresErrorHandler(),
+                        private val serializer: Serializer = MontaguSerializer.instance)
 {
-    private val logger = LoggerFactory.getLogger(ErrorHandler::class.java)
-    private val postgresHandler = PostgresErrorHandler()
+    private val unhandledExceptionMessage = "An unhandled exception occurred"
 
     init
     {
         sparkException<MontaguError>(this::handleError)
```

Comment:
I feel like this won't work any more, as only `MontaguError`'s will get caught, not any `Exception`?

Reply:
Oh yes, you're totally right, good catch